src/ontology/cl-edit.owl

```diff
@@ -26480,6 +26481,7 @@ EquivalentClasses(obo:CL_2000072 ObjectIntersectionOf(obo:CL_2000008 ObjectSomeV
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:TermGenie"^^xsd:string) obo:IAO_0000115 obo:CL_2000073 "Any migratory neural crest cell that is part of a cardiac neural crest."^^xsd:string)
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_2000073 <https://www.wikidata.org/entity/Q35563349>)
 AnnotationAssertion(oboInOwl:creation_date obo:CL_2000073 "2014-11-05T01:18:43Z"^^xsd:string)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_2000073 "cardiac neural crest cell")
```

Comment:
consider adding dbxref from [your comment](https://github.com/obophenotype/cell-ontology/issues/350#issuecomment-1201768685) to support this.

Reply:
good idea, done

<img width="260" alt="image" src="https://user-images.githubusercontent.com/6722114/182432568-8fc3b3fc-71be-436a-94ec-7538c571694a.png">